components/tooltip/Tooltip.js

```diff
@@ -28,203 +28,201 @@ const SIZES = {
   },
 };
 
-const tooltipFactory = () => {
-  return ComposedComponent => {
-    class TooltippedComponent extends Component {
-      static propTypes = {
-        children: PropTypes.node,
-        className: PropTypes.string,
-        onClick: PropTypes.func,
-        onMouseEnter: PropTypes.func,
-        onMouseLeave: PropTypes.func,
-        tooltip: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
-        tooltipColor: PropTypes.oneOf(['white', 'neutral', 'mint', 'violet', 'ruby', 'gold', 'aqua', 'inverse']),
-        tooltipHideOnClick: PropTypes.bool,
-        tooltipIcon: PropTypes.element,
-        tooltipPosition: PropTypes.oneOf(Object.keys(POSITIONS).map(key => POSITIONS[key])),
-        tooltipShowOnClick: PropTypes.bool,
-        tooltipSize: PropTypes.oneOf(Object.keys(SIZES)),
-        documentObject: PropTypes.object.isRequired,
+const Tooltip = ComposedComponent => {
+  class TooltippedComponent extends Component {
```

Comment:
I guess this can also be a PureComponent since shouldComponentUpdate is not being used? Or is this still needed because you were using forceUpdate?